src/views/Pages/Page404/Page404.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Codacy found an issue: ['React' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5714105)